engine/src/execution_graph/logic_controllers/taskflow/kernel.cpp

```diff
@@ -6,17 +6,18 @@ namespace ral {
 namespace cache {
 
 kernel::kernel(std::size_t kernel_id, std::string expr, std::shared_ptr<Context> context, kernel_type kernel_type_id)
-    : total_input_bytes_processed{0}, expression{expr}, kernel_id(kernel_id), kernel_type_id{kernel_type_id}, context{context} {
-    parent_id_ = -1;
-    has_limit_ = false;
-    limit_rows_ = -1;
-
-    logger = spdlog::get("batch_logger");
-    cache_events_logger = spdlog::get("cache_events_logger");
-
-    std::shared_ptr<spdlog::logger> kernels_logger;
-    kernels_logger = spdlog::get("kernels_logger");
-
+        : total_input_bytes_processed{0},
+          expression{expr},
+          kernel_id(kernel_id),
+          parent_id_(-1),
+          kernel_type_id{kernel_type_id},
+          context{context},
+          has_limit_(false),
+          limit_rows_(-1),
+          logger(spdlog::get("batch_logger")),
+          events_logger(spdlog::get("events_logger")) {
```

Comment:
`events_logger` no longer exists. please remove this line

Reply:
Done